demo/image-carousel-demo-app.html

```diff
@@ -0,0 +1,31 @@
+<link rel="import" href="../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../image-carousel.html">
+<link rel="import" href="../bower_components/polymer/lib/elements/dom-repeat.html">
+<dom-module id="image-carousel-demo-app">
+  <template>
+    <style>
+    </style>
+    <div>
+        <image-carousel id="my-carousel" aria-label="actual image-carousel">
+            <template slot="images" is="dom-repeat" items='[[images]]'>
+                <img class="scrolling-images" src="[[item]]" alt$="demo alt [[index]]" width="100px" height="350px">
+            </template>
+        </image-carousel>
+    </div>
+  </template>
+
+  <script>
+    class ImageCarouselDemoApp extends Polymer.Element {
+      static get is() { return 'image-carousel-demo-app'; }
+      static get properties() {
+        return {
+          // used to pass in images for the dom-repeat
+          images: {
+            type: Array
+            }
+          }
+      }
+    }
+    window.customElements.define(ImageCarouselDemoApp.is, ImageCarouselDemoApp);
+  </script>
+</dom-module>
```

Comment:
Can you fix indentation of this file also

Reply:
done.